test/integrationtests/skills/skill_tester.py

```diff
@@ -140,7 +140,7 @@ def load_skills(emitter, skills_root):
         from mycroft.util.log import LOG as skills_log
         buf = StringIO()
         with temporary_handler(skills_log, StreamHandler(buf)):
-            skill_loader = SkillLoader(emitter, path)
+            skill_loader = SkillLoader(emitter, path, None)
```

Comment:
The skill loader only takes two arguments.  How does this work?

Reply:
Good catch, it probably doesn't. will fix

Reply:
Fixed